src/commands.rs

```diff
@@ -4484,6 +4492,21 @@ fn cmd_git_remote_remove(
     Ok(())
 }
 
+fn cmd_git_remote_list(
+    ui: &mut Ui,
+    command: &CommandHelper,
+    _args: &GitRemoteListArgs,
+) -> Result<(), CommandError> {
+    let workspace_command = command.workspace_helper(ui)?;
+    let repo = workspace_command.repo();
+    let git_repo = get_git_repo(repo.store())?;
+    for remote_name in git_repo.remotes()?.iter().flatten() {
```

Comment:
Why is `flatten()` needed here?

Reply:
Because the iterator before it yields `Option<&str>`, so this `flatten()` skips the empty values. I had actually initially written it as an `if let Some(...)` but then [this Clippy lint](https://rust-lang.github.io/rust-clippy/master/#manual_flatten) told me to change to `flatten()`. I also find it unclear, but maybe we'll prefer it after we've been exposed to it a bit? Or we can disable that Clippy lint if you prefer (it would be the first one we disable).